### PR TITLE
Decrease and center ContextSubMenuArrow

### PR DESCRIPTION
**Pull request by joehenry9498**
## Changes
- Decrease arrow size and center it in ContextSubMenu

## Testing
- I tested the changes by running the storybook and seeing how the ContextMenu and ContextSubMenu looked in there.

Had to create a new pull request for this because there were git problems with the CLA

resolves https://github.com/iTwin/appui/issues/426

### CODE REVIEW

File: ui/core-react/src/core-react/contextmenu/ContextMenu.scss [hunk 1]
```diff
@@ -221,6 +221,8 @@ $uicore-menu-top-bottom-padding: calc(#{$uicore-menu-padding} * 0.6667);
         font-size: calc(var(--iui-font-size-2) * 0.5);
         line-height: var(--iui-size-l);
         text-align: center;
+        position: relative;
```
Comment (raplemie) [inline ui/core-react/src/core-react/contextmenu/ContextMenu.scss:224]: From what I see, most of the tweaks from this class were for the previous "webfont" mode, I think we can simplify all of this now by removing everything but 
```scss
{
  margin-left: $uicore-unbordered-padding;
  margin-top: var(--iui-size-3xs);
}
```
and get the same result with less "What's going on" when looking at this class.

**Review by raplemie — CHANGES_REQUESTED** [review]

File: ui/core-react/src/core-react/contextmenu/ContextMenu.scss (hunk 1 above)
Comment (joehenry9498) [reply ui/core-react/src/core-react/contextmenu/ContextMenu.scss]: Looks good in storybook with that simplified styling. Changes pushed

**Comment by raplemie** [comment]
Please add an entry in NextVersion.md (should be really similar to what was entered in the changes, but we will likely release that as part of 4.4.0 and NextVersion is the release document in this case.

**Review by raplemie — APPROVED** [review]
LGTM! Thanks!

**Pull request merged**